sjb/generated/test_pull_request_openshift_ansible_logging_36.xml

```diff
@@ -0,0 +1,801 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
@jcantrill can you post a diff between this file and the `sjb/generated/test_pull_request_openshift_ansible_logging.xml` normal one?

Reply:
https://gist.github.com/jcantrill/e3cf106aabd92e5f6dacf7785cf8692a